.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to ssf-desktop-wrapper
+:+1: First off, thanks for taking the time to contribute! :+1:
+
+# Contributor License Agreement (CLA)
+A CLA is a document that specifies how a project is allowed to use your
+contribution; they are commonly used in many open source projects.
+
+**_All_ contributions to _all_ projects hosted by the
+[Symphony Software Foundation](https://symphony.foundation/) must be made with
+a [Foundation CLA](https://symphonyoss.atlassian.net/wiki/display/FM/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement)
+in place, and there are [additional legal requirements](https://symphonyoss.atlassian.net/wiki/display/FM/Legal+Requirements) that must also be met.**
+
+As a result, PRs submitted to the ssf-desktop-wrapper project cannot be accepted until you have a CLA in place with the Foundation.
+
+# Contributing Issues
+
+## Prerequisites
+
+* [ ] Have you [searched for duplicates](https://github.com/ScottLogic/ssf-desktop-wrapper/issues?utf8=%E2%9C%93&q=)?  A simple search for exception error messages or a summary of the unexpected behaviour should suffice.
+* [ ] Are you running the [latest release of the ssf-desktop-wrapper](https://github.com/ScottLogic/ssf-desktop-wrapper/commits/master)?
+
+## Raising an Issue
+* Create your issue [here](https://github.com/ScottLogic/ssf-desktop-wrapper/issues/new).
+* New issues contain two templates in the description: bug report and enhancement request. Please pick the most appropriate for your issue, and delete the other.
+  * Please also tag the new issue with either "Bug" or "Enhancement".
```

Comment:
minor point - unless you have write access to a repo you cannot label issues, which is a bit of a pain. Labeling is unfortunately a triage task!